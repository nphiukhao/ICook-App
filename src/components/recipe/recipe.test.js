import React from 'react'

import Recipe from './recipeWParamsId'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'


const routeProps = {
    match: {
        params: {
            id: 1
        }
    }
}

describe(`Recipe component`, () => {
    it('Recipe in recipeWParamsId snapshot test', () => {
        const tree = shallow(<Recipe {...routeProps} />)
        expect(toJson(tree)).toMatchSnapshot()
    })
})

