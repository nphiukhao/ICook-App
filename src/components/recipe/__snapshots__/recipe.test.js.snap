// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe component Recipe in recipeWParamsId snapshot test 1`] = `
<div
  className="recipe-container"
>
  <h2>
     
  </h2>
  <div>
    <p>
      <span>
        Ingredients:
      </span>
       
    </p>
    <p>
      <span>
        Instructions:
      </span>
       
    </p>
    <span>
      Total Cooking Time: 
    </span>
  </div>
  <div
    className="recipe-buttons"
  >
    <button
      className="edit button"
      onClick={[Function]}
    >
      Edit
    </button>
    <button
      className="remove button"
      onClick={[Function]}
    >
      Remove
    </button>
  </div>
</div>
`;
